plugins/rtmp-services/data/services.json

```diff
@@ -1068,6 +1068,28 @@
                 "max video bitrate": 6000,
                 "max audio bitrate": 160
             }
+        },
+        {
+            "name": "Garena Live",
+            "common": true,
```

Comment:
Garena.tv is no common Service please remove it

Reply:
OK, I already removed it, thank you